Library/Formula/open-scene-graph.rb

```diff
@@ -62,7 +62,14 @@ def install
     end
 
     if build.with? "collada-dom"
-      args << "-DCOLLADA_INCLUDE_DIR=#{Formula["collada-dom"].opt_include}/collada-dom"
+      # https://github.com/Homebrew/homebrew/issues/43536
+      collada_include_candidates = Dir.glob("#{Formula["collada-dom"].opt_include}/collada-dom*")
+      if collada_include_candidates.empty?
+        odie "Could not locate collada-dom include directory"
+      end
+      collada_include_dir = collada_include_candidates[0]
```

Comment:
That looks more readable. I switched it to use this form, and `raise` instead of `odie`.